source/indexes/create-search-index.txt

```diff
@@ -31,8 +31,11 @@ To create an Atlas Search index, your deployment must be either:
 
 Additionally, your deployment must run MongoDB version 7.0 or later.
 
-Steps
------
+Procedures 
```

Comment:
I think we could probably keep this one as Steps to align with the task page prototype

Reply:
These are not sequential steps, but separate/disparate (create, modify, remove) actions that the user can take on the index. I think Procedures is correct in this context.